assets/base/i18n/en.json

```diff
@@ -3525,5 +3525,8 @@
   "webrtc.unmute_audio": "Unmute microphone",
   "webrtc.unpause_video": "Turn on camera",
   "webrtc.unsupported": "{username} client does not support video calls.",
-  "youtube_video.notFound": "Video not found"
+  "youtube_video.notFound": "Video not found",
+  "archived": "Archived",
```

Comment:
are these mobile specific? if so you need to add a `mobile.` prefix to them, if not these should be defined in the webapp

Reply:
Both of these are already defined in the webapp. I guess I can delete them here then.